File: input/java/Car01.java
class Vehicle01 {
    protected String brand = "Ford";        // Vehicle attribute
    public void honk() {                    // Vehicle method
        System.out.println("Tuut, tuut!");
    }
}

class Car01 extends Vehicle01 {
    private final String modelName = "Mustang";    // Car attribute
    public static void main(String[] args) {

        // Create a myCar object
        Car01 myCar = new Car01();

        // Call the honk() method (from the Vehicle class) on the myCar object
        myCar.honk();

        // Display the value of the brand attribute (from the Vehicle class) and the value of the modelName from the Car class
        System.out.println(myCar.brand + " " + myCar.modelName);
    }
}

File: input/kotlin/Car01.kt
import kotlin.jvm.JvmStatic

internal open class Vehicle01 {
    protected var brand = "Ford" // Vehicle attribute
    fun honk() {                    // Vehicle method
        println("Tuut, tuut!")
    }
}

internal class Car01 : Vehicle01() {
    private val modelName = "Mustang" // Car attribute

    companion object {
        @JvmStatic
        fun main(args: Array<String>) {

            // Create a myCar object
            val myCar = Car01()

            // Call the honk() method (from the Vehicle class) on the myCar object
            myCar.honk()

            // Display the value of the brand attribute (from the Vehicle class) and the value of the modelName from the Car class
            println(myCar.brand + " " + myCar.modelName)
        }
    }
}